### DISCUSSION TITLE
Term 'statamic' unrecognized in PowerShell after successfully installing Statamic CLI

### DISCUSSION THREAD

**Discussion by jayrdi** [statamic/cms#5822] [Q&A] [answered]
In PowerShell I have installed the Statamic CLI using "composer global require statamic/cli" with no errors but when I enter "statamic" or "statamic list" I just get "the term 'statamic' is not recognized as a name of a cmdlet...".

Does anyone know how to diagnose the issue?

Thanks

**Comment by JohnathonKoster** [comment, marked as answer]
Sounds like you might need to add Composer's `bin` directory to the the `PATH` variable. This answer on Stack Overflow has pretty good steps to do this (the `AppData` and related folders might be hidden, in case you don't see them right away):

https://stackoverflow.com/a/28393808

The answer states you should restart your PC - simply closing the PowerShell window and re-opening it after making the change should be enough

**Reply by jayrdi** [reply]
That did it, thanks!
